src/Components/Form/AddFormButton.js

```diff
@@ -1,4 +1,4 @@
-import ButtonAdd from '../UI/Button/AddButton.styles';
+import ButtonAdd from '../UI/Form/Button/AddButton.styles';
 
 export default function AddFormButton({ onSetFormButton }) {
```

Comment:
Another comment, sorry 😉. I think you would make your life easier if you'd simply use the `ButtonAdd` styled component directly on your `pages.js`. There's no need here to wrap the styled-component in another component (AddFormButton).

Reply:
It seems you deleted the `AddFormButton` component and decided for a more straight forward approach. Nice!